deployment/ccip/changeset/v1_6/cs_chain_contracts_test.go

```diff
@@ -859,8 +859,12 @@ func TestUpdateNonceManagersCSApplyPreviousRampsUpdates(t *testing.T) {
 				},
 			},
 		}),
-		testhelpers.WithNumOfChains(3),
-		testhelpers.WithChainIDs([]uint64{chainselectors.GETH_TESTNET.EvmChainID}))
```

Comment:
Chaanged to directly specify the chains instead of using a combination of N chains and direct selectors